main.nf

```diff
@@ -0,0 +1,684 @@
+
+    script:
+    """
+    FusionInspector \\
+        --fusions ${summary_fusions} \\
+        --genome_lib ${reference} \\
+        --left_fq ${reads[0]} \\
+        --right_fq ${reads[1]} \\
+        --CPU ${task.cpus} \\
+        --out_dir . \\
+        --out_prefix finspector \\
+        --prep_for_IGV
+    """
+}
+
+/*************************************************************
+ * Building report
+ ************************************************************/
+
+/*
+ * Parse software version numbers
+ */
+process get_software_versions {
+
+    when:
+    !params.test
+
+    output:
+    file 'software_versions_mqc.yaml' into software_versions_yaml
+
+    script:
+    """
+    echo $workflow.manifest.version > v_pipeline.txt
+    echo $workflow.nextflow.version > v_nextflow.txt
+    fastqc --version > v_fastqc.txt
+    multiqc --version > v_multiqc.txt
+    cat $baseDir/tools/fusioncatcher/Dockerfile | grep "VERSION" > v_fusioncatcher.txt
+    cat $baseDir/tools/fusion-inspector/environment.yml | grep "fusion-inspector" > v_fusion_inspector.txt
+    cat $baseDir/tools/star-fusion/environment.yml | grep "star-fusion" > v_star_fusion.txt
+    cat $baseDir/tools/ericscript/environment.yml | grep "ericscript" > v_ericscript.txt
+    cat $baseDir/tools/pizzly/environment.yml | grep "pizzly" > v_pizzly.txt
+    cat $baseDir/tools/squid/environment.yml | grep "squid" > v_squid.txt
+    scrape_software_versions.py > software_versions_mqc.yaml
+    """
+}
+
+/*
+ * FastQC
+ */
+process fastqc {
+    tag "$name"
+    publishDir "${params.outdir}/fastqc", mode: 'copy',
+        saveAs: {filename -> filename.indexOf(".zip") > 0 ? "zips/$filename" : "$filename"}
+
+    when:
+    !params.test
+
+    input:
+    set val(name), file(reads) from read_files_fastqc
+
+    output:
+    file "*_fastqc.{zip,html}" into fastqc_results
+
+    script:
+    """
+    fastqc -q $reads
+    """
+}
+
+/*
+ * MultiQC
+ */
+process multiqc {
+    tag "$name"
+    publishDir "${params.outdir}/MultiQC", mode: 'copy'
+
+    when:
+    !params.test
+
+    input:
+    set val(name), file(reads) from read_files_multiqc
+    file multiqc_config
+    file ('fastqc/*') from fastqc_results.collect()
+    file ('software_versions/*') from software_versions_yaml
+    file workflow_summary from create_workflow_summary(summary)
+    file fusions_mq from summary_fusions_mq.ifEmpty('')
+
+    output:
+    file "*multiqc_report.html" into multiqc_report
+    file "*_data"
+
+    script:
+    rtitle = custom_runName ? "--title \"$custom_runName\"" : ''
+    rfilename = custom_runName ? "--filename " + custom_runName.replaceAll('\\W','_').replaceAll('_+','_') + "_multiqc_report" : ''
+    """
+    create_mqc_section.py -i ${fusions_mq} -s "${name}"
+    multiqc -f $rtitle $rfilename --config $multiqc_config .
+    """
+}
+
+/*
+ * Output Description HTML
+ */
+process output_documentation {
+    publishDir "${params.outdir}/Documentation", mode: 'copy'
+
+    when:
+    !params.test
```

Comment:
I'm a bit confused by some of your `when` test conditionals... 😅

Reply:
If `params.test` is true ignore default nextflow processes.